### PR TITLE
styling and tests

### PR DESCRIPTION
**Pull request by kxp790**
US002 ready for review

### CODE REVIEW

**Review by rhysbarrett — APPROVED** [review]

File: capabilities-db/mysql.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[mysql]
```
Comment (aowenmeehanwork) [inline capabilities-db/mysql.config:1]: This shouldn't be in the repo!!! Make sure you add "mysql.config" into the .gitignore

File: e2e/src/detail-viewer-role.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export class DetailViewerRole {
+    navigateTo() {
+        return browser.get(browser.baseUrl += 'detail-viewer/role/1') as Promise<any>;
+    }
+    getTitle() {
+        return element(by.xpath('//detail-viewer/div/div/mat-card')).getText() as Promise<string>;
+    }
+    getJobSummaryText() {
+        element(by.xpath('//*[@id="mat-expansion-panel-header-0"]/span[1]/mat-panel-title')).click();
+        delay(2000);
```
Comment (aowenmeehanwork) [inline e2e/src/detail-viewer-role.ts:13]: Is there any reason a manual delay is here? Are you waiting for an event, or is this some sort of simulation for loading?

File: src/app/detail-viewer/detail-viewer.component.css [hunk 3]
```diff
@@ -5,4 +5,9 @@
 
 .main-detail-content {
     text-align: left;
-}
\ No newline at end of file
+}
+
+p {
+    text-align: left;
+    margin-right: 30px;
```
Comment (aowenmeehanwork) [inline src/app/detail-viewer/detail-viewer.component.css:12]: Can this be done with percentages at all?

**Review by aowenmeehanwork — COMMENTED** [review]
Review attached

**Pull request closed without merging**